### ISSUE TITLE
 stable/grafana Unistall plugins

### ISSUE THREAD

**Issue by Lakshmisowmya** [helm/charts#14373]
**Describe the bug**

**Which chart**:
Grafana

**What happened**:
Unable to remove installed plugins 

**What you expected to happen**:
Uninstall plugins in grafana

**How to reproduce it** (as minimally and precisely as possible):
1.Install plugins by making changes in values.yaml.
2.Plugin gets installed.
3. Remove the entry in values.yaml
4.Plugin do not get removed from grafana.

**Anything else we need to know**:

If removing the plugin from values.yaml is not the right way to uninstall plugins. Provide a way to uninstall them. 
I accidentally installed a plugin that is not being supported by  grafana version that i am using and now the pod do not come up with the plugin.json file missing issue.

I tried to unistall the plugin but seems there is no way to do it.  Pod do not come back at all.

**Comment by JordanYefet12** [comment]
Is there any solution for this?

**Comment by ArunaLakmal** [comment]
I hit with the same issue and anything to resolve this? Damn, my working Grafana is no longer running.

**Comment by ArunaLakmal** [comment]
> I hit with the same issue and anything to resolve this? Damn, my working Grafana is no longer running.

I had the similar situation with Kubernetes and I was struggling to bring my pod up and running. I deployed through the helm and removing plugin didn't work out for me.

I manually edited the deployment like this:

spec: template: spec: containers: - grafana command: ["grafana-cli"] args: ["plugins", "remove", "grafana-image-renderer"]

Then the container failed again with below message:
Removing plugin: grafana-image-renderer Error: ✗ plugin does not exist

Then I removed the updated commands in the deployment and it worked out for me.
